unattended_installer/builder.sh

```diff
@@ -0,0 +1,219 @@
+#!/bin/bash
+
+# Tool to create wazuh_install.sh, wazuh-cert-tool.sh
+# and wazuh-passwords-tool.sh
+# Copyright (C) 2015, Wazuh Inc.
+#
+# This program is a free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation.
+
+base_path="/home/verdx/Documents/Wazuh/entorno/wazuh-packages/unattended_installer"
```

Comment:
Please change base path

Reply:
Changed in eb83cc7ec604c15342e4fe987303b0347a6d0d9b